tests/providers/google/cloud/operators/test_automl.py

```diff
@@ -379,63 +379,51 @@ def test_templating(self, create_task_instance_of_operator):
         assert task.impersonation_chain == "impersonation-chain"
 
 
-class TestAutoMLListColumnsSpecsOperator:
+class TestAutoMLTablesListColumnsSpecsOperator:
+    expected_deprecation_warning = (
+        "Class `AutoMLTablesListColumnSpecsOperator` has been deprecated and will be removed after 31.12.2024. "
+        "AutoML platform no longer supports tabular datasets after its integration with Cloud Translation and Vertex AI. "
+        "Please refer to https://cloud.google.com/vertex-ai/docs/start/migrating-to-vertex-ai and "
+        "https://cloud.google.com/translate/docs/advanced/automl-upgrade for more info about available alternatives."
+    )
+
     @mock.patch("airflow.providers.google.cloud.operators.automl.CloudAutoMLHook")
     def test_execute(self, mock_hook):
         table_spec = "table_spec_id"
         filter_ = "filter"
         page_size = 42
 
-        op = AutoMLTablesListColumnSpecsOperator(
-            dataset_id=DATASET_ID,
-            table_spec_id=table_spec,
-            location=GCP_LOCATION,
-            project_id=GCP_PROJECT_ID,
-            field_mask=MASK,
-            filter_=filter_,
-            page_size=page_size,
-            task_id=TASK_ID,
-        )
-        op.execute(context=mock.MagicMock())
-        mock_hook.return_value.list_column_specs.assert_called_once_with(
-            dataset_id=DATASET_ID,
-            field_mask=MASK,
-            filter_=filter_,
-            location=GCP_LOCATION,
-            metadata=(),
-            page_size=page_size,
-            project_id=GCP_PROJECT_ID,
-            retry=DEFAULT,
-            table_spec_id=table_spec,
-            timeout=None,
-        )
```

Comment:
Why this part is removed from tests?

Reply:
Because `AirflowException` is raised in the operator's `__init__` method. Therefore we cannot instantiate it and run any methods.